tests/unit/ReportSecondaryActionUtilsTest.ts

```diff
@@ -311,10 +321,10 @@ describe('getSecondaryAction', () => {
             statusNum: CONST.REPORT.STATUS_NUM.APPROVED,
         } as unknown as Report;
         const policy = {
-            connections: {[CONST.POLICY.CONNECTIONS.NAME.QBD]: {config: {export: {exporter: CURRENT_USER_EMAIL}}}},
```

Comment:
did we need to update the test case with `autoSync`? or we should've added another test and retain this one.

Reply:
Yes, I think we had to update it